server/ot_rpc.go

```diff
@@ -304,14 +324,25 @@ func (h *lphttp) TranscodeResults(w http.ResponseWriter, r *http.Request) {
 				break
 			}
 
-			encodedPixels, err := strconv.ParseInt(p.Header.Get("Pixels"), 10, 64)
-			if err != nil {
-				glog.Error("Error getting pixels in header:", err)
-				res.Err = err
-				break
+			if len(p.Header.Values("Pixels")) > 0 {
```

Comment:
I think this if/else if block will result in either the pixel count being read from the response or the phash being read from the response. Instead, we should read the pixel count from the response if it is available and the phash if it is available.